tests/shared/src/main/scala/munit/scalacheck/ScalaCheckFrameworkSuite.scala

```diff
@@ -0,0 +1,34 @@
+package munit
+package scalacheck
+
+import org.scalacheck.Prop.forAll
+import org.scalacheck.rng.Seed
+
+class ScalaCheckFrameworkSuite extends ScalaCheckSuite {
+
+  override def scalaCheckTestParameters =
+    super.scalaCheckTestParameters.withInitialSeed(Seed(123L))
```

Comment:
this is needed for making the test output stable for the failed test. It also server as a test for overriding these parameters

Reply:
👍

Reply:
might be worth commenting in the source instead